hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/impl/ContainerSet.java

```diff
@@ -252,6 +252,21 @@ public Iterator<Container<?>> getContainerIterator(HddsVolume volume) {
         .iterator();
   }
 
+  /**
+   * Get the number of containers based on the given volume.
+   *
+   * @param volume hdds volume.
+   * @return number of containers
+   */
+  public long getContainerCount(HddsVolume volume) {
```

Comment:
```suggestion
  public long containerCount(HddsVolume volume) {
```

Then we can change the existing `containerCount` method to return a long, making the two variants overloads.